src/StdAfterTest.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+// Inheritted by test contracts where afterTest() is needed
+abstract contract StdAfterTest {
```

Comment:
We will need to inherit from this here: https://github.com/foundry-rs/forge-std/blob/4e5f46a8014eb0a40526e88af6c89001096e4656/src/Test.sol#L30